axelrod/strategies/graaskamp.py

```diff
@@ -0,0 +1,122 @@
+from axelrod.action import Action, actions_to_str
+from axelrod.player import Player
+from axelrod.random_ import randrange
+from axelrod._strategy_utils import calculate_scores
+
+from enum import Enum
+
+C, D = Action.C, Action.D
+
+class Mode(Enum):
```

Comment:
Can we remove this class and just use a string as the corresponding attribute in the player class.